java/test/src/main/java/test/Ice/idleTimeout/Client.java

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) ZeroC, Inc.
+
+package test.Ice.idleTimeout;
+
+public class Client extends test.TestHelper {
+  @Override
+  public void run(String[] args) {
+    var properties = createTestProperties(args);
+    properties.setProperty("Ice.Package.Test", "test.Ice.idleTimeout");
```

Comment:
We can avoid setting `Ice.Package` for tests that don't use classes.

Reply:
Yes, but should we?

I think the pattern is "always set Ice.Package.xxx". No need to check if we define any Slice class or exception in the associated Slice.